.project

```diff
@@ -10,8 +10,14 @@
 			<arguments>
```

Comment:
This file shouldn't be checked in. Actually, as part of the mavenization tasks we are removing IDE specific files all together.